src/ChannelServer/World/WorldManager.cs

```diff
@@ -9,6 +9,7 @@ namespace Melia.Channel.World
 	public class WorldManager
 	{
 		private int _handles = 0;
+		private int _buffhandles = 0;
```

Comment:
Do buffs use their own handle id range, starting at 0? Or do they go into the same pool as the others?

Reply:
So similar to those skill "handles" or skill actor ids, they seem to be independent of entity handles.
The player/character handle is 379706 and the Buff "handle" was 331185, next buff "handle" I got was about a minute later was 331846, a minute later it was 332232. So it looks like it overlaps with regular entity handles.

Reply:
Sound like it. Alright 👍